test/unit/refred/reduction_table_handling/test_update_reduction_table.py

```
@@ -147,5 +146,28 @@ def test_clear_row_when_both_data_and_norm_runs_cleared(mock_file_finder_find_ru
             assert item.text() == ""
 
 
+@mock.patch("refred.calculations.locate_list_run.FileFinder.findRuns")
+def test_run_with_missing_properties(mock_file_finder_find_runs, qtbot, file_finder_find_runs, data_server):
```

Ruff (0.14.10)

Unused function argument: `data_server`

(ARG001)